hudi-common/src/main/java/org/apache/hudi/common/table/timeline/ArchivedTimelineLoader.java

```diff
@@ -40,6 +40,7 @@ public interface ArchivedTimelineLoader extends Serializable {
   public void loadInstants(
       HoodieTableMetaClient metaClient,
       @Nullable HoodieArchivedTimeline.TimeRangeFilter filter,
+      HoodieArchivedTimeline.LogFileFilter logFileFilter,
```

Comment:
Let's add a new method named `loadInstantsV1` and keep this one unchanged.

Reply:
Good point.  Fixed.